README.md

```diff
@@ -30,9 +30,9 @@ export default class SomeComponent extends React.Component {
       fetchSomeData.subscribe( data => this.setstate({data}) )
     );
   }
-  componentWillReceiveProps(nextProps) {
+  componentDidUpdate(prevProps) {
```

Comment:
so we don't embarrass ourselves with something as foolish as componentWillReceiveProps.